tests/results/ResultFilterTest.py

```diff
@@ -368,6 +368,72 @@ def test_affected_code(self):
                                                        new_result_list)),
                                  sorted(unique_new_result_list))
 
+    def test_affected_code_rename_files(self):
+
+        # ORIGINAL SOURCE RANGES:
+        sr0_pre_change = SourceRange.from_values("file_name",
+                                                 start_line=4,
+                                                 start_column=1,
+                                                 end_line=4,
+                                                 end_column=6)
+
+        # ORIGINAL RESULTS:
+        res0_pre_change = Result(origin="origin",
+                                 message="message",
+                                 affected_code=(sr0_pre_change,))
+
+        # NEW SOURCE RANGES:
+        sr1_pre_change = SourceRange.from_values("file_name_new",
+                                                 start_line=4,
+                                                 start_column=1,
+                                                 end_line=4,
+                                                 end_column=6)
+        sr1_change = SourceRange.from_values("file_name_new",
+                                             start_line=4,
+                                             start_column=8,
+                                             end_line=4,
+                                             end_column=13)
+
+        # NEW RESULTS:
+        res1_pre_change = Result(origin="origin",
+                                 message="message",
+                                 affected_code=(sr1_pre_change,))
+        res1_change = Result(origin="origin",
+                             message="message",
+                             affected_code=(sr1_change,))
+        res1_whole_remove = Result.from_values(origin="origin",
+                                               message="message",
+                                               file="file_name_new",
+                                               line=6,
+                                               column=1,
+                                               end_line=7,
+                                               end_column=5)
+
+        original_result_list = [res0_pre_change]
+
+        new_result_list = [res1_pre_change,
+                           res1_change,
+                           res1_whole_remove]
+
+        unique_new_result_list = [res1_pre_change,
```

Comment:
ah the line changes, but your testcase doesn't really test this, does it? You should rather add a result that stays the same due to the renaming, hm? Make a result in line 7 (8 in the original file)! That one's shifted and renamed, if we can detect that properly that should work out fine.